app/api/contact/specs/contact.spec.js

```diff
@@ -36,5 +37,30 @@ describe('contact', () => {
         })
         .catch(catchErrors(done));
     });
+    it('should send email with the provided sender email and site name', async done => {
+      const settings = {
+        site_name: 'some site name',
+        senderEmail: 'sender@email.com',
+      };
+      const set = await settingsModel.get();
+      const updated = Object.assign(set[0], { ...settings });
+      await settingsModel.save(updated);
+      contact
+        .sendMessage({
+          email: 'bruce@wayne.com',
+          name: 'Bruce Wayne',
+          message: 'I want to contact you.',
+        })
+        .then(() => {
+          expect(mailer.send).toHaveBeenCalledWith({
+            from: `"${settings.site_name}" <${settings.senderEmail}>`,
+            subject: 'Contact mesage from Bruce Wayne bruce@wayne.com',
+            text: 'I want to contact you.',
+            to: 'contact@uwazi.com',
+          });
+          done();
+        })
+        .catch(catchErrors(done));
+    });
```

Comment:
pls use async/await instead of ```done``` for new code, and when you can also refactor old ones.